docs/src/pages/style/links/links.md

```diff
@@ -0,0 +1,25 @@
+---
+components: Link
+---
+
+# Links
+
+<p class="description">The Link component allows you to show links on the page, open them in a new window.</p>
+
+## Simple links
+
+{{"demo": "pages/style/links/Links.js"}}
+
+The Link component is built on top of the [Typography](/api/typography/) component.
+You can leverage its properties.
```

Comment:
This is slightly confusing, as the demo that follows doesn't actually uses the properties of Typography, but rather wraps it in the Typography component, while the demo thet precees it does.